CommonMark.Portable45/CommonMark.Portable45.csproj

```diff
@@ -0,0 +1,175 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I was unable to achieve that. Is it a GUID that has to be added to
ProjectTypes?
On Nov 24, 2015 12:38, "Kārlis Gaņģis" <notifications@github.com> wrote:

> In CommonMark.Portable45/CommonMark.Portable45.csproj
> <https://github.com/Knagis/CommonMark.NET/pull/56#discussion_r45719097>:
>
> > @@ -0,0 +1,175 @@
> > +﻿<?xml version="1.0" encoding="utf-8"?>
>
> the project file should use the exising Base project so that the files are
> not included in more that one place.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Knagis/CommonMark.NET/pull/56/files#r45719097>.
>